forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -269,6 +269,7 @@ public class Card extends GameEntity implements Comparable<Card>, IHasSVars {
     private Map<Integer, Integer> damage = Maps.newHashMap();
     private boolean hasBeenDealtDeathtouchDamage;
     private boolean hasBeenDealtExcessDamageThisTurn;
```

Comment:
Can this variable be removed if excessDamageThisTurnAmount > 0?

Reply:
I wondered about that... I think so. I was just worried about the LKI part in CardDamageMap line 131

Reply:
yea the logic for _Rith, Liberated Primeval_ probably needs to be refactored for that to work:
> if a creature or planeswalker an opponent controlled was dealt excess damage this turn

Reply:
Maybe leave both for now and see what WotC does next? They seem to be experimenting with "excess damage" right now.